Classes/Event/EventBus.php

```diff
@@ -1,54 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\Cqrs\Event\Exception\EventBusException;
-use Neos\Cqrs\EventListener\EventListenerLocator;
-use TYPO3\Flow\Annotations as Flow;
-use TYPO3\Flow\Log\SystemLoggerInterface;
-use TYPO3\Flow\Utility\TypeHandling;
-
-/**
- * @Flow\Scope("singleton")
- */
-class EventBus
```

Comment:
Well, as stated, the EventBus is the single point of infrastructure to swap out to support a different pub/sub transport. I think we introduced the EventPublisher only to somehow separate the process of publishing, but I don't think that was actually a good idea. Basically we partially renamed the EventBus to EventPublisher... so if that's the case, I'm fine with getting rid of either one.